CondTools/L1TriggerExt/plugins/L1CondDBPayloadWriterExt.cc

```diff
@@ -1,21 +1,59 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-#include "CondTools/L1TriggerExt/plugins/L1CondDBPayloadWriterExt.h"
```

Comment:
Should this header be removed?

Reply:
I'd let the owners of the code do that, as it's not really needed for this migration

Reply:
To me it would be least confusing if the PR moving contents of a header to a source file would also remove the now-obsolete header.

Reply:
oh yeah, I'll do it

Reply:
Done in https://github.com/cms-sw/cmssw/pull/36383/commits/e9025cb558e1849300f0aa9e4a31ee9547e1b54f

Reply:
@tvami 
It's not clear to me why this modification has been done.
why to remove the header?